blocks/basic/test/qa_Selector.cpp

```diff
@@ -55,7 +55,9 @@ struct repeated_source : public gr::Block<repeated_source<T>> {
     }
 };
 
-ENABLE_REFLECTION_FOR_TEMPLATE_FULL((typename T), (repeated_source<T>), identifier, remaining_events_count, values, out);
+ENABLE_REFLECTION_FOR_TEMPLATE(repeated_source, identifier, remaining_events_count, values, out);
+
+// NOTE: If you move repeated_source to blocklib, call GR_REGISTER_BLOCK
```

Comment:
please remove these and other related comments. Either refactor this specific blocks so that they can be part of the official blocklib or drop the comment because it's a superfluous documentation with its meaning soon being forgotten.

N.B. we agreed that unit-tests should -- in large -- not rely on custom block definitions that are only available in the unit-tests but rely on the official curated ones from the blocklib. This makes maintenance, refactoring, and co-use of common functionality so much easier.

Also, many of this unit-test-only blocks have redundant functions to the existing blocklibs ones.